packages/react-components/src/components/ActionMenu/constants.tsx

```diff
@@ -57,4 +57,9 @@ export const exampleOptions = [
     ),
     onClick: noop,
   },
+  ...[...Array(10)].map((_, index) => ({
+    key: `option${index + 8}`,
+    element: <ActionMenuItem>{`Menu item #${index + 8}`}</ActionMenuItem>,
+    onClick: noop,
+  })),
```

Comment:
We need to separate those storybook options from our constants.ts file, because after the build the whole file ends up in output files, and we do not wont to have storybook examples in the final build